demo-cloud-migration.md

```diff
@@ -0,0 +1,199 @@
+~~~
+
+This command initiates 10 concurrent client workloads for 20 minutes, but limits each worker to 100 operations per second (since you're running everything on a single machine).
+
+## Step 4. Watch data balance across all 3 nodes
+
+Open the Admin UI at `http://localhost:8080` and hover over the **SQL Queries** graph at the top. After a minute or so, you'll see that the load generator is executing approximately 95% reads and 5% writes across all nodes:
+
+<img src="images/admin_ui_sql_queries.png" alt="CockroachDB Admin UI" style="border:1px solid #eee;max-width:100%" />
+
+Scroll down a bit and hover over the **Replicas per Node** graph. Because CockroachDB replicates each piece of data 3 times by default, the replica count on each of your 3 nodes should be identical:
+
+<img src="images/admin_ui_replicas_migration.png" alt="CockroachDB Admin UI" style="border:1px solid #eee;max-width:100%" />
+
+## Step 5. Add 3 nodes on "cloud 2"
+
+Again, the flag to note is [`--locality`](configure-replication-zones.html#descriptive-attributes-assigned-to-nodes), which you're using to specify that these next 3 nodes are running on "cloud platform 2".
+
+{{site.data.alerts.callout_info}}If you were running nodes across clouds for real, you'd also configure firewalls to allow inbound and outbound communication between all the nodes. {{site.data.alerts.end}}
+
+~~~ shell
+# In a new terminal, start node 4 on cloud 2
+$ cockroach start --insecure \
+--locality=cloud=platform2 \
+--store=cloud2node4 \
+--host=localhost \
+--port=26261 \
+--http-port=8083 \
+--join=localhost:26257 \
+--cache=100MB
+
+# In a new terminal, start node 5 on cloud 2:
+$ cockroach start --insecure \
+--locality=cloud=platform2 \
+--store=cloud2node5 \
+--host=localhost \
+--port=25262 \
+--http-port=8084 \
+--join=localhost:26257 \
+--cache=100MB
+
+# In a new terminal, start node 6 on cloud 2:
+$ cockroach start --insecure \
+--locality=cloud=platform2 \
+--store=cloud2node6 \
+--host=localhost \
+--port=25263 \
+--http-port=8085 \
+--join=localhost:26257 \
+--cache=100MB
+~~~
+
+## Step 6. Watch data balance across all 6 nodes
+
+Back in the Admin UI, hover over the **Replicas per Node** graph again. Because you set [`--locality`](configure-replication-zones.html#descriptive-attributes-assigned-to-nodes) settings to specify that nodes are running across 2 clouds, you'll see an approximately even replica count on each node, indicating that CockroachDB has automatically rebalanced replicas across the cloud platforms:
+
+<img src="images/admin_ui_replicas_migration2.png" alt="CockroachDB Admin UI" style="border:1px solid #eee;max-width:100%" />
+
+## Step 7. Migrate all data to "cloud 2"
+
+In a new terminal, edit the default replication zone, adding a [hard constraint](configure-replication-zones.html#replication-constraints) that all replicas must be on nodes with `--locality=cloud=platform2`:
```

Comment:
Why don't you provide some colorful additional narration to these steps that are demonstrating the value proposition. For example: "OK, we're now running three nodes one cloud 1. But what if we'd like to start experimenting with resources provided by another cloud vendor? Let's try that by adding three more nodes to a new cloud platform."

"We now have the cluster replicating across our two simulated cloud vendors. But after experimentation, we're happy with cloud vendor 2, and we decide that we'd like to move everything there. Can we do that without interruption to our live client traffic? Yes, and it's as simple as adding a constraint to our replication configuration that all replicas must be chosen from cloud vendor 2."

You get the idea...we need to give people a scenario so it's not just an abstraction that only folks who've been struggling specifically with these problems will understand.